py/kubeflow/kubeflow/ci/access_management_tests.py

```diff
@@ -0,0 +1,43 @@
+""""Argo Workflow for testing Access Management's OCI image using Kaniko"""
```

Comment:
Same herer: 

I feel like you can abstract this Builder python file given you have multiple almost same file here, add it in e.g, py/kubeflow/kubeflow/ci/base_builder.py

And for each Argo Workflow, just reference it should be fine

Reply:
I'm not sure it makes sense for these CI files, as I am assuming they will be extended with more tests similar to https://github.com/kubeflow/kubeflow/blob/master/py/kubeflow/kubeflow/ci/common_ui_tests.py

Reply:
For extensibility, I would suggest using parent class as base, and child class for patch.

Reply:
@kimwnasptd What is your thought on this?

Reply:
@PatrickXYS to make sure I'm synced with the proposed changes; the argument is to create a common base class for building a workflow that always has a step for kaniko-build, with `--no-push`, since we essentially use it in all of the component CI files?

Reply:
@kimwnasptd I'm also not completely clear what the proposed implementation would look like. Either way, I'm not sure if this is a blocker as it testing if images can be built seems like something that should be added sooner rather than later. This way we would have discovered that the admission-webhook was broken before https://github.com/kubeflow/kubeflow/pull/5661.
Also, https://github.com/kubeflow/kubeflow/issues/5482#issuecomment-791906596 still needs to be resolved. Should the registry be called `access-management` or should it be `kfam`?

Reply:
This is only NIT change, if we feel it's urgent, feel free to move forward

Reply:
I think this can be dealt with in the future when the tests for each component get expanded. For now, I think it is most important images start getting built and the PRs that are being merged for the release are at least tested if they break the image building.

Reply:
Regarding the changes for using a different class I agree with @DavidSpek that this isn't a blocker, so let's go with this implementation for now and if in the future we find ourselves repeating too much code then lets go over this argument.
Does this sound OK to you @PatrickXYS ?

Reply:
> Should the registry be called access-management or should it be kfam?

@yanniszark what are your thoughts on this? I guess we should stick to `access-management` for now?

I'll also give a heads up to our tracking issue to add this registry as well

Reply:
@kimwnasptd I actually just ran into an issue where using the common `kaniko_builder.py` actually complicates things slightly. For this situation specifically it is easily fixed, however; for the CI testing I think it could just complicate things with little added value.

Reply:
> @kimwnasptd I actually just ran into an issue where using the common kaniko_builder.py actually complicates things slightly. For this situation specifically it is easily fixed

Could you provide some more details for this? Just so that we can identify the pattern/pain-point once we see it reoccurring

Reply:
>  I guess we should stick to access-management for now?
@kimwnasptd I would vote for `kfam`, as it is what we usually call the component.

Reply:
Also kfam actually stands for Kubeflow Access Management (just like we don't call it kubeflow-pipelines, but kubeflow/pipelines)

Reply:
@yanniszark @kimwnasptd I think this discussion should move to https://github.com/kubeflow/kubeflow/issues/5675. Then this PR can be merged and I will update all the registry naming depending on the outcome of https://github.com/kubeflow/kubeflow/issues/5675.